todo.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script type="text/javascript">
+</script>
+</head>
+
+<body>
+<h1>Todo List</h1>
+    <div id="todoList"></div>
+<hr>
+    <input type="text" id="todoDescription" placeholder="Add a todo here"/>
+    <button id="addTodo">Add</button>
+    
+    <div>Remaining tasks:<a id="remainingCount">0</a></div>
+    <div>Completed tasks:<a id="completedCount">0</a></div>
+<script >
+
+var totalTasks = 0;
+
+function addToList(todoItem, todoDescription) {
+    var newTodo = document.createElement("div");
+    var todoCheckbox = document.createElement("input");
+    todoCheckbox.setAttribute('type', 'checkbox');
+    todoCheckbox.setAttribute('name', 'oneCheckbox');
+    todoCheckbox.setAttribute('onchange','changeHandler()');    
+    newTodo.appendChild(todoCheckbox);
+    newTodo.innerHTML = newTodo.innerHTML + todoDescription + '  ';
+    
+
+    var removeTodoBtn = document.createElement("button");
+    var btnText = document.createTextNode("Remove");
+    removeTodoBtn.appendChild(btnText);
+
+    newTodo.appendChild(removeTodoBtn);
```

Comment:
Good use of whitespace, you logically break down things that belong together by using vertical space, which makes it easier to read.